ShowOrchard.Theme/Views/Layout.liquid

```diff
@@ -21,7 +21,7 @@
 
     {% render_section "HeadMeta", required: false %}
 
-    {% shape "GoogleTag", GoogleTagPropertyId: "G-C8EEHMLQLP", CookieDomain: "auto" %}<!-- #spell-check-ignore-line -->
+    {% google_tag property_id: "G-C8EEHMLQLP", cookie_domain: "auto" %}<!-- #spell-check-ignore-line -->
```

Comment:
Shouldn't this display something locally too? The template's changes show up for me, but this displays nothing.

Reply:
Summoning @sarahelsaig, because these are her changes from the `task/system-text-json-migration` branch.

Reply:
No, it never displayed anything outside of production mode. This was true [when it was in Lombiq.BaseTheme](https://github.com/Lombiq/Orchard-Base-Theme/pull/122/files#diff-736f0d6f936f738922808c24b7f8ee8d530338dcfd6f53c30efc0eb1e7ce188aL14) and didn't change [when the GoogleTag shape was moved into HE](https://github.com/Lombiq/Helpful-Extensions/commit/3f312f60f2129b26cf4fb434358ef3550b31a67d#diff-2dcab13645aaab146c25b77b298995e1e985e2ebc80f344927e82523c3e460baR14) (btw both `{% google_tag %}` for Liquid and `<google-tag>` for Razor are just friendly wrappers around the shape tag helper with pre-defined arguments).

Reply:
OK then!